cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/SlangTextualKeys.java

```diff
@@ -29,10 +29,13 @@
     String ON_FAILURE_KEY = "on_failure";
 
     //action
-    String JAVA_ACTION = "java_action";
-
-    //operation
-    String ACTION_KEY = "action";
+    String JAVA_ACTION_KEY = "java_action";
+    String JAVA_ACTION_CLASS_NAME_KEY = "class_name";
+    String JAVA_ACTION_METHOD_NAME_KEY = "method_name";
+    String JAVA_ACTION_GAV_KEY = "gav";
```

Comment:
I take it back :)
I understand we are going to use Maven inside CloudSlang as the optional mechanism for retrieving artifacts. In that case, `gav` is the correct term